doc/THIRD-PARTY-LICENSES

```diff
@@ -13,3 +13,4 @@ src/tools/website-frontend/resources/assets/skin/base/fonts/font-awesome/font-aw
 src/tools/website-frontend/resources/assets/skin/base/util/space.less: mit
 src/tools/web/client/public/Roboto-Light.ttf: apache-2.0
 src/tools/web/client/public/Roboto-Regular.ttf: apache-2.0
+src/tools/kdb/gen/mustache.hpp: BSL-1.0
```

Comment:
Thank you for maintaining! I made it now mandatory in f256c0dbc8f6c9189aff2e8ae7f20bdfe9c4f16b